test/Functional/CreateTokenTest.php

```diff
@@ -27,4 +27,14 @@ public function testCreateTokenPermanentlyFails()
 
         $this->porter->importOne(new ImportSpecification(new CreateToken(FixtureFactory::createInvalidCard())));
     }
+
+    /**
+     * Tests that when a token cannot be created because of an non chargeable card.
```

Comment:
This comment doesn't seem to make sense. If we remove *when* it makes some sense but we also need to state the expected outcome. Perhaps we could write, *Tests that when a token is created for a non-chargeable card an unrecoverable exception is thrown.*